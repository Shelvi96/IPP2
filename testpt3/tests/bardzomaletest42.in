NEW ID

78>088:222:14
:31452515:>593193
9>1
0;;6618:8:>17;4
41:>6;
;07897967>626
248029755;>71610:
4611128926>915519;;
84453635>2:30381
823254>::901
5:517>;;52535
:5>721;46309
:97:69>74
;>9145:9338;
1227968389>73;
3>6234
795948>7:2909
::>774637862
@9
@7900:80;4:15224506402:39
@14515886004943138:9;323
@6348:173904714118::153
@22519086:4:7246
@5:20;;2
@70385;4;524261845;
@;719004133
@8:;4;837889027
@20:17:09;11364
@5
@94580038051206445607:16;2:
@:42;1689;85;0:02
@77745;;30
@763440781543850852804773
