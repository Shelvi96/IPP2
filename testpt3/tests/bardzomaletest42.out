0
877637953440
28531181591
1100120000
111
0
122100
0
1
12
0
126379865295360
1110
0
35298311949
